postgres.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/library/postgres:13
+
+# Set correct ownership and permissions to cert and key it needs to be
+# done via dockerfile becuase simply mounting the files as volumes
+# with the postgres ownership tampers with other containers
+# See: https://stackoverflow.com/questions/55072221/deploying-postgresql-docker-with-ssl-certificate-and-key-with-volumes
```

Comment:
I don't think we need to do this, in fact the stack overflow post mentions that this is only necessary for windows, in linux permissions are kept between the host and container systems and in the `generate_local_pg_tls_cert` `make` entrypoint we already set the permissions correctly (0640 for postgres user/group), if this is not working then maybe we need to set the UID/GUID explicitly for the cert and key, which is apparently 999 [1]

[1] https://stackoverflow.com/questions/55072221/deploying-postgresql-docker-with-ssl-certificate-and-key-with-volumes

Reply:
This is what I initially though that would be enough. But 0640 wasn't enough:

```2022-08-25 09:05:38.354 UTC [43] DETAIL:  File must have permissions u=rw (0600) or less if owned by the database user, or permissions u=rw,g=r (0640) or less if owned by root```

It didn't work with the root, so I changed to the 0600 and postgres (999). Setting the UID/GUID explicitly of the cert and key to 999 ultimately failed all the containers with the:

```Error: lsetxattr /home/jfrejlac/work/osidb/etc/pg/local-server.crt: operation not permitted```

So in the end, the only way which worked for me was setting the chmod and chown in the Dockerfile and this also didn't tampered with the other containers.

Reply:
Yeah that's a SELinux issue, I managed to fix it by doing `podman unshare chown 999:999 etc/pg/local-server.*`, I will share with you my local diff, should probably add it to the docs or do it as part of the generate local tls make entrypoint, and chmod to 0600

Reply:
I added code from your local diff and used that command and it works. Added it also to the Makefile rule which handles the pg tls certs so the ownership and the persmissions should now be correctly set and everything works as it should. If you agree, I can finally merge this.